packages/api/src/index.ts

```diff
@@ -107,7 +113,7 @@ passport.use(
 );
 
 void (async () => {
-  const orm = await initDatabase();
+  orm = await initDatabase();
```

Comment:
If we aren't using this elsewhere, let's remove the `let` definition up top and make it a `const` here
```suggestion
  const orm = await initDatabase();
```

Reply:
We are using it on line around line 141 for the contribution poll.  It needs access to an entity manager